src/lib/ec2Handler.js

```diff
@@ -0,0 +1,43 @@
+import { EC2Client, StopInstancesCommand, StartInstancesCommand } from "@aws-sdk/client-ec2";
+import core from "@actions/core";
+import { sleep, consoleTable } from "./utils.js";
+
+
+const client = new EC2Client();
+
+export async function start(InstanceIds) {
+try {
+    core.info(`Starting instances: ${InstanceIds}`)
+    const command = new StartInstancesCommand({InstanceIds: InstanceIds})
+    let response
+    response = await client.send(command);
+
+    // TODO -Implement a loop to check the status of the instances until they are in the desired state
+    // Suggest to not use describeInstanceStatus apisince it requires permission too open.
+    // Wait for 5 seconds and send the request again to retrieve the status of the instances
+    await sleep(5000);
+    response = await client.send(command);
+    consoleTable(response.StartingInstances)
```

Comment:
I'm using `console.table` because I'm so lazy If we use `core.info` we need to create the talbe itself, Just if we want to use an ASCII table to list the instance status.

Reply:
There's actually a way of doing that without reimplementing the wheel using the `node:console` API. But, I got you, that's fine for now.